File: SEPADirectDebit/src/main/java/com/braintreepayments/api/sepadirectdebit/SEPADirectDebitPaymentAuthResultInfo.java
package com.braintreepayments.api.sepadirectdebit;

import com.braintreepayments.api.BrowserSwitchFinalResult;

/**
 * Details of a {@link SEPADirectDebitPaymentAuthResult.Success}
 */
public class SEPADirectDebitPaymentAuthResultInfo {

    private BrowserSwitchFinalResult.Success browserSwitchSuccess;

    SEPADirectDebitPaymentAuthResultInfo(BrowserSwitchFinalResult.Success browserSwitchSuccess) {
        this.browserSwitchSuccess = browserSwitchSuccess;
    }

    BrowserSwitchFinalResult.Success getBrowserSwitchSuccess() {
        return browserSwitchSuccess;
    }

}

File: SEPADirectDebit/src/main/java/com/braintreepayments/api/sepadirectdebit/SEPADirectDebitPaymentAuthResultInfo.kt
package com.braintreepayments.api.sepadirectdebit

import com.braintreepayments.api.BrowserSwitchFinalResult

/**
 * Details of a [SEPADirectDebitPaymentAuthResult.Success]
 */
data class SEPADirectDebitPaymentAuthResultInfo(
    val browserSwitchSuccess: BrowserSwitchFinalResult.Success
)
